fab/fabfile.py

```diff
@@ -32,7 +32,9 @@
 import sh
```

Comment:
this is how we'd authenticate for github and be able to create tags: https://github.com/dimagi/commcare-hq/pull/11292/commits/2765c60b0ca4acabd529ae58d88aa146bbe9a0d1

easiest to just setup an api key, otherwise it'll just ask for your username and password